ext/mixed/css/display.css

```diff
@@ -180,7 +180,7 @@ h2 {
     filter: grayscale(100%);
 }
 
-.action-next>img {
+.action-button.action-next:before {
```

Comment:
Same needed for at least:

- `.navigation-header:not([data-has-previous=true]) .navigation-header-actions .action-previous>img,
.navigation-header:not([data-has-next=true]) .navigation-header-actions .action-next>img {`
- `.actions .disabled img {`

Currently the buttons aren't grayed out when disabled.

Reply:
bcd1055183958dc915108f47b303f4ca99a03bd7